code/modules/mob/living/carbon/alien/diona/diona_nymph.dm

```diff
@@ -4,6 +4,7 @@
 
 //Diona time variables, these differ slightly between a gestalt and a nymph. All values are times in seconds
 /mob/living/carbon/alien/diona
+	desc = "The most basic form of dionea."
```

Comment:
```suggestion
	desc = "The most basic form of diona."
```

Reply:
Is there anything else you think needs fixing?